docs/integrations/mlstudio_notebooks.md

```diff
@@ -0,0 +1,93 @@
+# Azure Machine Learning Notebooks Integration
+
+Connecting to the Feature Store from Azure Machine Learning Notebooks requires setting up a Feature Store API key for Azure Machine Learning Notebooks and installing the **HSFS** on the notebook. This guide explains step by step how to connect to the Feature Store from Azure Machine Learning Notebooks.
```

Comment:
the **HSFS** python library on the

Reply:
Is there no way to define lots of the repetitive text in other .md files and include them in another file?
I am thinking about the "create an API key" section and maybe others - like Network Connectivity below.

Reply:
I think there is, we did it in the old docs but there is so much repetition that it should be a separate effort.